src/modules/dashboard/components/CreateColonyWizard/StepCreateToken.tsx

```diff
@@ -190,17 +183,6 @@ const StepCreateToken = ({
                 help={MSG.helpTokenSymbol}
               />
             </div>
-            <div className={styles.inputFieldWrapper}>
-              <ActionFileUpload
-                name="tokenIcon"
```

Comment:
If we removed the Token info from the native token creation screen, shouldn't we also remove it from the external token "creation" page?:

https://github.com/JoinColony/colonyDapp/blob/2f02621bbb7fda94b9d452b49647609660abc64e/src/modules/dashboard/components/CreateColonyWizard/StepSelectToken.tsx#L187-L197

Reply:
Also, I think we need to make @JoinColony/product aware of this change

Reply:
Yes, we should remove that, good catch!